client-react/src/pages/app/functions/function/function-log/FunctionLog.tsx

```diff
@@ -161,6 +163,10 @@ const FunctionLog: React.FC<FunctionLogProps> = props => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [logLevel, allLogEntries]);
 
+  const getLogsMessage = () => {
+    return showFilteredLogsMessage ? t('connectedAndFilteredMessage') : t('connected');
+  };
```

Comment:
Consider using the `useCallback` hook since this function depends on `showFilteredLogsMessage`.

```suggestion
  const getLogsMessage = React.useCallback(() => {
    return showFilteredLogsMessage ? t('connectedAndFilteredMessage') : t('connected');
  }, [showFilteredLogsMessage]);
```